test/native/integration-test-helpers/rich-text-type.js

```diff
@@ -17,29 +17,26 @@ function insertTextAtPosition( text, newText, start, end ) {
  * Changes the text and selection of a RichText component.
  *
  * @param {import('react-test-renderer').ReactTestInstance} richText                        RichText test instance.
- * @param {string}                                          text                            Text to be set.
+ * @param {string}                                          text                            Text to set.
  * @param {Object}                                          options                         Configuration options for selection.
- * @param {number}                                          [options.initialSelectionStart]
- * @param {number}                                          [options.initialSelectionEnd]
- * @param {number}                                          [options.selectionStart]        Selection start position.
- * @param {number}                                          [options.selectionEnd]          Selection end position.
+ * @param {number}                                          [options.initialSelectionStart] Selection start position before the text is inserted.
+ * @param {number}                                          [options.initialSelectionEnd]   Selection end position before the text is inserted.
+ * @param {number}                                          [options.selectionStart]        Selection start position after the text is inserted.
+ * @param {number}                                          [options.selectionEnd]          Selection end position after the text is inserted.
```

Comment:
For clarity, I'm thinking about renaming `selectionStart` and `selectionEnd` to `finalSelectionStart` and `finalSelectionEnd` as these are the final selection values after the text is inserted, WDYT?